tox.ini

```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist = py36, py37, py38
+
+[flake8]
+max-line-length = 99
+
```

Comment:
We don't really need this flake8 section here after having removed the other flake8 stuff, though it probably doesn't hurt anything to stay if you are going to add flake8 check back soon in another PR.